sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/LocalRelation.scala

```diff
@@ -73,6 +73,6 @@ case class LocalRelation(output: Seq[Attribute], data: Seq[InternalRow] = Nil)
     }
   }
 
-  override lazy val statistics =
+  override def statistics(implicit parents: Option[Seq[LogicalPlan]] = None): Statistics =
```

Comment:
Why would you add the implicit parents? Where do you need these?

Also Option[Seq[..]] is kinda an anti-pattern, since you can pass an empty Seq around.